radio/src/gui/128x64/view_statistics.cpp

```diff
@@ -117,6 +117,7 @@ void menuStatisticsView(event_t event)
   #define MENU_DEBUG_COL2_OFS          (17*FW)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove the whitespaces before the # character. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3APPDirectiveIndentation)